### PR TITLE
Seed w images

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: db/migrate/20220823095256_user_id_to_bike.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UserIdToBike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :bikes, :user, foreing_key: true, null: false
```
Comment (onofrecc) [inline db/migrate/20220823095256_user_id_to_bike.rb]: typo... foreign_key

File: app/models/bike.rb [hunk 2]
```diff
@@ -8,8 +8,8 @@ class Bike < ApplicationRecord
     # end
 
 
-  CATEGORIES = ['City bike', 'Road bike', 'Mountain bike']
-  GEARS = ['3', '7', '18', '21', '28']
+  CATEGORIES = ['City bike', 'Road bike', 'BMX bike', 'Cargo bike', 'Mountain bike', 'Single-speed bike', 'Fixie bike']
+  GEARS = ['1', '3', '7', '18', '21', '28']
```
Comment (onofrecc) [inline app/models/bike.rb]: First element of the array could be replaced by a string..

'No gears'

**Comment by A-drienne** [comment]
Looks good!

Idea for next steps: 
If we want quickly make bikes of the different categories (bmx etc), you could create one bike of each category (with image, description etc) using Bike.new. That way we can then work on the filters

File: db/seeds.rb [hunk 3]
```diff
@@ -22,20 +31,178 @@
...
+#     v1661180398/FreddieFiets/fixie2_e5fm5x.png
+#     v1661180397/FreddieFiets/fixie1_b1frvw.png
+#   ]
+#   10.times do
+#     bike = Bike.new(
+#       name: Faker::Music::RockBand.name,
+#       bike_type: ['Single-speed bike', 'Fixie bike'].sample,
+#       location: Faker::Address.city,
+#       description: Faker::Lorem,
+#       gears: '1',
```
Comment (onofrecc) [inline db/seeds.rb]: No gears

File: db/migrate/20220823095256_user_id_to_bike.rb (hunk 1 above)
Comment (ghost) [reply db/migrate/20220823095256_user_id_to_bike.rb]: done

File: app/models/bike.rb (hunk 2 above)
Comment (ghost) [reply app/models/bike.rb]: done

**Pull request merged**